test/screenshot/lib/report-generator.js

```diff
@@ -149,24 +149,20 @@ class ReportGenerator {
           <td class="report-metadata__cell report-metadata__cell--val">${numTestCases}</td>
         </tr>
         <tr>
-          <th class="report-metadata__cell report-metadata__cell--key">Git User:</th>
+          <th class="report-metadata__cell report-metadata__cell--key">User:</th>
```

Comment:
I love all of this (the golden commit, snapshot commit, etc) 💜

Reply:
Thanks!! 😀